src/logic/application-code/dotenv/onesdk-log.ts

```diff
@@ -0,0 +1,2 @@
+export const ONESDK_LOG_ENV = 'ONESDK_LOG';
+export const ONESDK_LOG_COMMENT = 'Set to "on" to enable better output logging';
```

Comment:
nitpick: I feel like “enable better logging” implies the default logging is not good. Based on [OneSDK's host docs](https://github.com/superfaceai/one-sdk/blob/247dceed182c93c9dd008b173879e009f7dd376b/host/javascript/README.md?plain=1#L149) I'd suggest the following description

```suggestion
export const ONESDK_LOG_COMMENT = 'Set to "on" to enable debug logging';
```